clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -69,6 +71,7 @@
 public class ConfigDef {
 
     public static final Object NO_DEFAULT_VALUE = new String("");
+    public static final String EXECUTABLE_PASSWORD_ENABLE_CONFIG = "executable.password.enable";
```

Comment:
Can you explain your reasoning to have 1 config that makes all passwords an executable? Instead of adding individual configs like "ssl.keystore.password.generator"?